src/java/fr/paris/lutece/plugins/blog/business/BlogAdminDashboardHome.java

```diff
@@ -0,0 +1,46 @@
+package fr.paris.lutece.plugins.blog.business;
+
+import fr.paris.lutece.portal.service.spring.SpringContextService;
+import java.sql.Date;
+
+
+public final class  BlogAdminDashboardHome
```

Comment:
Could you please extract the value of id-dashboard into a parameter so that the code can be reused later for creating other dashboards